src/effect.cpp

```diff
@@ -469,14 +469,23 @@ std::string effect::disp_desc(bool reduced) const
     std::vector<desc_freq> values;
     // Add various desc_freq structs to values. If more effects wish to be placed in the descriptions this is the
     // place to add them.
-    values.push_back(desc_freq(get_percentage("PAIN", reduced), get_avg_mod("PAIN", reduced), _("pain"), _("pain")));
-    values.push_back(desc_freq(get_percentage("HURT", reduced), get_avg_mod("HURT", reduced), _("damage"), _("damage")));
-    values.push_back(desc_freq(get_percentage("THIRST", reduced), get_avg_mod("THIRST", reduced), _("thirst"), _("quench")));
-    values.push_back(desc_freq(get_percentage("HUNGER", reduced), get_avg_mod("HUNGER", reduced), _("hunger"), _("sate")));
-    values.push_back(desc_freq(get_percentage("FATIGUE", reduced), get_avg_mod("FATIGUE", reduced), _("fatigue"), _("rest")));
-    values.push_back(desc_freq(get_percentage("COUGH", reduced), get_avg_mod("COUGH", reduced), _("coughing"), _("coughing")));
-    values.push_back(desc_freq(get_percentage("VOMIT", reduced), get_avg_mod("VOMIT", reduced), _("vomiting"), _("vomiting")));
-    values.push_back(desc_freq(get_percentage("SLEEP", reduced), get_avg_mod("SLEEP", reduced), _("blackouts"), _("blackouts")));
+    int val = 0;
+    val = get_avg_mod("PAIN", reduced);
+    values.push_back(desc_freq(get_percentage("PAIN", val, reduced), val, _("pain"), _("pain")));
```

Comment:
Huh.  This looks like it might be workable for the mutation fix.  I wasn't sure how best to handle that.